webknossos/webknossos/geometry/vec3_int.py

```diff
@@ -135,7 +135,8 @@ def __add__(self, other: Union[int, "Vec3IntLike"]) -> "Vec3Int":  # type: ignor
     def __sub__(self, other: Union[int, "Vec3IntLike"]) -> "Vec3Int":
         return self._element_wise(other, sub)
 
-    def __mul__(self, other: Union[int, "Vec3IntLike"]) -> "Vec3Int":
+    # note: (arguments incompatible with superclass, do not multiply Vec3Int with a plain tuple! Hence the type:ignore)
```

Comment:
This is due to a different reason, multiplying Vec3Int with a tuple works:
```python
>>> import webknossos as wk
>>> wk.Vec3Int(3, 2, 3) * (3, 2, 1)
Vec3Int(9,4,3)
```

```suggestion
    # Note: When multiplying regular tuples with an int those are repeated,
    # which is a different behavior in the superclass! Hence the type:ignore.
```